packages/xchain-client/src/BaseXChainClient.ts

```diff
@@ -146,4 +148,5 @@ export abstract class BaseXChainClient implements XChainClient {
   abstract getTransactions(params?: TxHistoryParams): Promise<TxsPage>
   abstract getTransactionData(txId: string, assetAddress?: string): Promise<Tx>
   abstract transfer(params: TxParams): Promise<string>
+  abstract transferOffline(params: TxOfflineParams): Promise<StdTx>
```

Comment:
Is `transferOffline` something we do need for all clients (xchain-btc|bch|eth|bnb etc.)? If you do need it just for xchain-cosmos|thorchain, extend the interface of clients in xchain-cosmos|thorchain - no need to introduce `transferOffline` in `BaseXChainClient.ts`.